examples/plot_titanic.py

```diff
@@ -0,0 +1,228 @@
+"""
+=======
+Titanic
+=======
+
+This example is based on `the similarly named Kaggle challenge <https://www.kaggle.com/c/titanic/overview>`_
+
+We will be training and testing with only one :term:`node`.
+
+Authors:
+  |    Romain Goussault, :fa:`github` `RomainGoussault <https://github.com/RomainGoussault>`_
+  |    Maria Telenczuk, :fa:`github` `maikia <https://github.com/maikia>`_
+
+"""
+
+# %%
+# Import all the dependencies
+# ---------------------------
+
+from pathlib import Path
+from io import BytesIO
+import os
+from urllib.request import urlopen
+import zipfile as zipfile
+
+# %%
+# You should have already Substra installed, if not follow the instructions here: :ref:`Installation`
+#
+
+import substra
+
+# %%
+# Next, we need to link to the already defined assets. You can download them from here:
+#
+# TODO: add a link and the instructions on how to get the assets folder (.zip??) and update the path if necessary
+#
+# TODO: in the assets directory you can find XXX files including datafiles which we will use next
+# TODO: a link to some example on how to prepare your own assets + each file explained??
+#
+
+url_assets = 'https://github.com/Substra/substra-documentation/blob/b38747fab3b7ba4cc9e95f511d0c761348ccaf65/code_examples/assets.zip?raw=true'
+http_response = urlopen(url_assets)
+zipedfile = zipfile.ZipFile(BytesIO(http_response.read()))
+zipedfile.extractall()
```

Comment:
`FileExistsError: [Errno 17] File exists: substra-documentation/examples/assets/dataset`

There is an error if we already built the docs once apparently

Reply:
I am unable to reproduce it. Could you please provide some info on how you got it?